src/mpio.jl

```diff
@@ -0,0 +1,61 @@
+using .MPI
```

Comment:
```suggestion
using MPI
```
?

Reply:
Should be `using .MPI`. That's just how Requires works.

From their [README](https://github.com/JuliaPackaging/Requires.jl/blob/master/README.md):

> In the @require block, or any included files, you can use or import the package, but note that you must use the syntax `using .Gadfly` or `import .Gadfly`, rather than the usual syntax. Otherwise you will get a warning about Gadfly not being in dependencies.